src/os/mainctrl.js

```diff
@@ -333,6 +333,12 @@ os.MainCtrl = function($scope, $element, $compile, $timeout, $injector) {
   os.areaImportManager.registerImportUI(os.file.mime.filter.TYPE, new os.filter.im.OSFilterImportUI());
   os.areaFileManager = new os.file.FileManager();
 
+  // initialize the places import/file managers
+  os.placesImportManager = new os.ui.im.ImportManager();
+  os.placesImportManager.registerImportDetails('Data filters for supported layers.');
```

Comment:
The description here looks like copy pasta. Please update it or remove it altogether.